### DISCUSSION TITLE
need help for deploying to GCP with terraform [502 error]

### DISCUSSION THREAD

**Discussion by theindieduo** [serverpod/serverpod#2196] [Q&A]
I am trying to deploy serverpod to GCP with terraform. 

Got all steps done. Github action runs smoothly. Terraform setup done. Able to connect to database through postico. Ran migration script. Just that when I hit [my api site](https://app.zerox.today/) I get 502. 

I don't know what's wrong here.

Any ideas?

Been a great fan of serverpod so far, spent 10 days learning and trying it in localhost. I just need to deploy it to be able to start building my side project. 

Disclaimer: Pure front end dev here, very little exposure to infra side of things. Might be a small issue.

**Comment by omjogani** [comment]
Hi @theindieduo 

Please ensure your instance running on GCP. If yes then connect to SSH run the command `docker ps` and ensure your docker container is running perfectly.

If you don't find any container running, you better look at the logs of the running instance. You might be facing an error in the log. Look for the solution to that error and try solving it.

Hopefully, it will solve your issue.
Thanks!
